.github/workflows/pull_request_title_check.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
```

Comment:
This url doesn't work any more. I've been redirected to the homepage
![image](https://github.com/bigcommerce/stencil-cli/assets/95914987/bd877db3-adc8-46e8-a0ee-048b3a0d58df)

Reply:
I will remove this section at all